cypress.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  retries: 1,
+  video: false,
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      // eslint-disable-next-line import/extensions
+      return require('./cypress/plugins/index.js')(on, config);
```

Comment:
Nit: in my other PR I moved the logic inside this function, as it's also what's [recommended by Cypress](https://docs.cypress.io/guides/references/migration-guide#Plugins-File-Removed).

https://github.com/commercetools/merchant-center-application-kit/pull/2713/files#diff-3f62e885350a48939f240b9daf40f6b06d41da7751f2b613238fc7669ac0a1b2R8-R33

It's also easier to use `async/await`.

Reply:
Thanks for pointing that out.

Updated in this commit: d0a488e8cc2ba865426d571c20b33fe0109a47ad